src/workflows/airqo_etl_utils/datautils.py

```
@@ -52,15 +54,16 @@ def extract_devices_data(
         local_file_path = f"/tmp/devices.csv"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)